definitions/hello-release/hello-release.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I wouldn't expect to see this file in a PR for adding a yaml linter. This should be in a different PR or completely removed as we don't really need this anymore.

Reply:
You can safely ignore this file, it's from the main branch, pulled in via a rebase.

This PR was rebased on main.
That is why you see superfluous commits from the main branch.
What I do is add multiple remotes in my local repo:

My fork remote looks like this:
```
$ git remote get-url origin
git@github.com:parasense/release-bundles.git
```

So I just add another remote for the upstream of my fork, and then rebase from that.
```
git remote add upstream https://github.com/release-service/release-bundles
git fetch upstream
git rebase upstream/main
git push origin/branch-name
```

Reply:
Rebase doesn't work like that. Please, fix this PR so it only contains the relevant stuff.